uploadNeuroDB/minc_insertion.pl

```diff
@@ -476,17 +477,17 @@ =head2 Methods
     \%studyInfo, 0, $centerID, $NewScanner, $upload_id
 );
 my $subjectIDsref = $utility->determineSubjectID(
-    $scannerID, \%studyInfo, 0, $upload_id
+    $scannerID, \%studyInfo, 0, $upload_id, $User, $centerID
 );
 
 
 
 
 
 ## Validate that the candidate exists and that PSCID matches CandID
-my $CandMismatchError = $utility->validateCandidate($subjectIDsref);
```

Comment:
removed this as it is now part of the determineSubjectIDs function and is returned into $subjectIDsref->{'CandMismatchError'}